kolibri/core/apps.py

```diff
@@ -31,6 +34,9 @@ def ready(self):
                 settings=os.environ["DJANGO_SETTINGS_MODULE"]
             )
         )
+        # When configured to use Redis, make sure we have sensible Redis settings
+        if CACHE_OPTIONS["CACHE_BACKEND"] == "redis":
```

Comment:
Can probably just always call this method, and put the conditional check inside the method itself?

Reply:
I'll update this to go along with other `CACHE_OPTIONS` change

Reply:
Sigh, moving this inside now makes the function "too complex" :expressionless:

Reply:
@bjester: Hey linter, how do you feel about this code?
Linter: It's complicated!

Reply:
I reduced it as much as I could, which wasn't enough, so I added the comment to ignore it.